packages/react-native-layout/src/types/generic-props.ts

```diff
@@ -0,0 +1,12 @@
+export interface TestableComponent {
+  testID?: string;
+}
+
+export interface PaddingOrMarginProps {
+  top?: number | string;
+  right?: number | string;
+  bottom?: number | string;
+  left?: number | string;
```

Comment:
don't use these as props directly, I found out the hard way. RN will add them as absolute offsets on top of your intended padding and mess with your brain fora few hours as your try to figure it out.
source: me for a few hours yesterday